bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rvec.py

```diff
@@ -96,17 +165,15 @@ def get_array_interface(self):
             else:
                 pointer = GetDataPointer(self, cppname, "data")
             return {
-                "shape": (size, ),
+                "shape": (size,),
                 "typestr": "{}{}{}".format(endianess, dtype_numpy, dtype_size),
                 "version": 3,
-                "data": (pointer, False)
+                "data": (pointer, False),
```

Comment:
These cosmetic changes could go a in a separate commit

Reply:
I have put the cosmetic changes in a separate commit